.github/workflows/maven.yml

```diff
@@ -15,13 +15,16 @@ jobs:
       matrix:
         java: [8, 11, 17]
         os: [ubuntu-latest, windows-latest, macos-latest]
+        include:
+          - os: 'macos-14'
```

Comment:
Super nitpicky, could we swap the order of the two items so that the display name in https://github.com/ome/bioformats/actions/runs/7909123358/job/21589620169 is consistent with the other builds ?

Reply:
Done in a07fcde.